services/short-url-manager/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.7'
+
+x-short-url-manager: &short-url-manager
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: signon
```

Comment:
I think the image needs changing here.

Reply:
D'oh! thanks for catching. My C+P source revealed to the world 😊